_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,431 @@
+
+
+### CAST/Type system enhancements
+
+Data appears in all sorts and shapes but is often not in the type that you need 
+it to be, which is why 
+[casting](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/types/#casting) 
+is one of the most common operations in SQL. In Flink 
+1.15, the default behavior of a failing CAST has changed from returning a null to 
+returning an error, which makes it more compliant with the SQL standard. The old 
+casting behavior can still be used by calling the newly introduced TRY_CAST function
+or restored via a configuration flag.
+
+In addition, many bugs have been fixed and improvements made to the casting 
+functionality, to ensure correct results. 
+
+
+### JSON functions
+
+JSON is one of the most popular data formats and SQL users increasingly need to build 
+and read these data structures.  Multiple 
+[JSON](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/functions/systemfunctions/#json-functions)
+functions have been added to Flink SQL 
+according to the SQL 2016 standard. It allows users to inspect, create, and modify JSON 
+strings using the Flink SQL dialect.
+
+
+## Community enablement
+
+Enabling people to build streaming data pipelines to solve their use cases is our goal. 
+The community is well aware that a 
+technology like Apache Flink is never used on its own and will always be part of a 
+bigger architecture. Thus, it is important that Flink operates well in the cloud, 
+connects seamlessly to other systems, and continues to support programming languages like Java and Python. 
+
+
+### Cloud interoperability
+
+There are users operating Flink deployments in cloud infrastructures from various 
+cloud providers. There are also services that offer to manage Flink deployments for 
+users on their platform. 
+
+In Flink 1.15, a recoverable writer for Google Cloud Storage has 
+been added. We also organized the connectors in the Flink ecosystem and put some focus 
+on connectors for the AWS ecosystem (i.e. 
+[KDS](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/kinesis/), 
+[Firehose](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/firehose/)).
+
+
+### The Elasticsearch sink
+
+There was significant work on Flink’s overall connector ecosystem, but we want to highlight the 
+[Elasticsearch sink](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/elasticsearch/) because it was implemented with 
+the new connector interfaces, which offers asynchronous functionality coupled with end-to-end semantics. 
+This sink will act as a template to follow in the future.
+
+
+### A Scala-free Flink
+
+A detailed 
+[blog post](https://flink.apache.org/2022/02/22/scala-free.html)  
+already explains the ins and outs of why Scala users can now use the Flink 
+Java API with any Scala version (including Scala 3). 
```

Comment:
Not sure why, but at least Github's markdown interpreter adds a line break after the link - maybe the double-space at the end of its line? Please check with what Hugo is rendering there...
```suggestion
A detailed 
[blog post](https://flink.apache.org/2022/02/22/scala-free.html) 
already explains the ins and outs of why Scala users can now use the Flink 
Java API with any Scala version (including Scala 3). 
```

Reply:
One more thing: I feel like we should highlight changes that users have to apply to their job dependencies.

Working on https://github.com/apache/flink-training/pull/48/files, I found these:
- `flink-clients_${scalaBinaryVersion}` ->`flink-clients`
- `flink-streaming-java_${scalaBinaryVersion}` -> `flink-streaming-java`
- `flink-runtime-web_${scalaBinaryVersion}` -> `flink-runtime-web`
- `flink-test-utils_${scalaBinaryVersion}` -> `flink-test-utils`

Reply:
i built the docs and it looks good, added a sentence about the dependencies.